tools/Makefile.am

```diff
@@ -19,55 +19,85 @@ SUBDIRS = . nut-scanner
 EXTRA_DIST = nut-usbinfo.pl nut-recorder.sh nut-ddl-dump.sh \
   gitlog2changelog.py nut-snmpinfo.py driver-list-format.sh
 
-all: nut-scanner-deps 
+GENERATED_SNMP_FILES = nut-scanner/nutscan-snmp.h
+
+GENERATED_USB_FILES = nut-scanner/nutscan-usb.h
+
+# Hotplug output file
+GENERATED_USB_FILES += ../scripts/hotplug/libhid.usermap
+
+# udev output file
+GENERATED_USB_FILES += ../scripts/udev/nut-usbups.rules.in
+
+# BSD devd output file
+GENERATED_USB_FILES += ../scripts/devd/nut-usb.conf.in
+
+# UPower output file
+GENERATED_USB_FILES += ../scripts/upower/95-upower-hid.rules
+
+all: nut-scanner-deps $(GENERATED_USB_FILES)
 
 # XXX these rules are all bogus!  They cause un-named target files to
```

Comment:
it looks like this comment is out-of-date, at least for this particular rule.

Reply:
The way this is rendered is a bit misleading - I meant that "XXX these rules are all bogus!" was out-of-date with respect to the rule below (which does have specific dependencies), not the green portion of the previous diff.

Reply:
Thanks for pointing out that I never got to it, I hoped to but forgot ;)